public/editor/scripts/ui/publisher.js

```diff
@@ -130,6 +131,50 @@ Publisher.prototype.showUnpublishedChangesPrompt = function(callback) {
   });
 };
 
+//Using generateRequest("unpublish") in order to sync description
+//throws error since nothing is being unpublished, however this works
+//to save description. This is currently a "hack", a more elegant
+//solution may be implemented after
+Publisher.prototype.setDescription = function() {
+  var publisher = this;
```

Comment:
Let's not force a "publish" just to save the description. Instead, let's make the following changes:

To this function:
```js
const publisher = this;
const oldDescription = Project.getDescription();
const description = publisher.dialog.description.val();

if(oldDescription === description) {
  return;
}

Project.setDescription(description);

const data = {
  title: Project.getTitle(),
  description,
  dateUpdated: new Date().toISOString()
};

Metadata.update({
  update: true,
  csrfToken: publisher.csrfToken,
  host: Project.getHost(),
  id: Project.getId(),
  data 
}, error => {
  console.error("[Thimble] Failed to update project description with: ", error);
});
```

Reply:
you'll also need to require in the `metadata` module from https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/scripts/project/metadata.js

Reply:
This does not to work, even with the metadata module included. After inspecting, it seems this is producing a bad request.

![capture](https://user-images.githubusercontent.com/14240261/31840077-ba7c2d3c-b5b1-11e7-984c-58cfec4d16d2.JPG)